src/components/SearchList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function SearchList({ albums }) {
+  return (
+    <ul className="album-list">
+      {albums.map((album) => (
+        <li key={album._id} className="album-item">
+          <img src={album.image} alt={album.albumName} className="album-image" />
+          <p className="album-title">{album.albumName}</p>
+          <p className="album-artist">{album.artistName}</p>
+        </li>
```

Comment:
This is fine to keep in-line as it is. Just wanted to call out that in the tickets, the idea was to have a separate component for the search items. The main advantage of this is the idea of "[Separating Concerns](https://dev.to/tamerlang/separation-of-concerns-the-simple-way-4jp2)" by making each component only have to be concerned about a specific thing. So `SearchList` is concerned with rendering the list and making sure the list is styled well (and maybe in the future things like pagination so that if we get back 100 albums, we're not showing _all_ of them at once). Then there could be a `SearchItem` component or `Album` component or something like that, that is specifically concerned with how to render a _single_ album:

```javascript
/* SearchList.js */

function SearchList({ albums }) {
  return (
    <ul className="album-list">
      {albums.map((album) => <SearchItem album={album} key={album._id} />)}
    </ul>
  );
}
```

```javascript
/* SearchItem.js */

function SearchItem({ album }) {
  return (
        <li className="album-item">
          <img src={album.image} alt={album.albumName} className="album-image" />
          <p className="album-title">{album.albumName}</p>
          <p className="album-artist">{album.artistName}</p>
        </li>
  );
}
```